eng/versioning/supported_external_dependency_versions.json

```diff
@@ -123,6 +127,18 @@
     { "com.fasterxml.jackson.module:jackson-module-parameter-names": "2.17.0" },
     { "com.fasterxml.jackson.module:jackson-module-scala_2.12": "2.17.0" }
   ],
+  "jackson_2.18":
```

Comment:
nit: consider naming this `jackson_2.18_rc`
```suggestion
  "jackson_2.18_rc":
```

Reply:
Following with previous patterns, I'd rather leave this as `2.18` and update the dependency versions accordingly when it goes GA.